packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -434,9 +430,8 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
       "signature",
     ];
     this.validateUserOp(userOp, requiredFields);
-    Logger.log("userOp validated");
     if (!this.bundler) throw new Error("Bundler is not provided");
-    Logger.log("userOp being sent to the bundler", userOp);
+    console.info("userOp being sent to the bundler", userOp);
```

Comment:
We're logging everything now, regardless of the environment variable right? I quite liked the logger. Did we only get rid of it because it was in the common package?

Reply:
Won't we be spamming browser consoles now?

Reply:
Currently moved to console for time being, as I wanted to remove the dependency of common package.

Need to raise another PR where we can either use the same Logger in utils or use another logger service. Keeped the console so we get what to repalce